data-prepper-plugins/http-source/README.md

```diff
@@ -27,7 +27,18 @@ source:
 
 ## Metrics
 
-TBD
+### Counter
+- `requestsReceived`: measures total number of requests received by `/log/ingest` endpoint.
+- `requestsRejected`: measures total number of requests rejected by HTTP source plugin.
+- `successRequests`: measures total number of requests successfully processed by HTTP source plugin.
+- `badRequests`: measures total number of requests with invalid content type or format processed by HTTP source plugin.
+- `requestTimeouts`: measures total number of requests that time out in the HTTP source server.
+
+### Timer
+- `requestProcessDuration`: measures latency of requests processed by the HTTP source plugin. 
+
+### Summary
+- `payloadSummary`: measures the distribution of incoming requests payload sizes.
```

Comment:
What is the unit here?

I am concerned we are coupling the underlying metrics framework with the current name. I was confused by a payloadSummary. Why not call it payloadSize? I think that is more intuitive to our users.